### PR TITLE
FAQ and more content documentation changes

### PR DESCRIPTION
**Pull request by anara123**
## Status
- [x] Ready
- [ ] In Progress
- [ ] In Hold

## Related Issues
https://github.com/demisto/etc/issues/20259

## Description
- Add FAQ documentation
- Improve Context and Outputs doc

### CODE REVIEW

File: docs/context_and_ouputs/README.MD [hunk 1]
```diff
@@ -1,126 +1,134 @@
...
-
 Context is a map (dictionary) /JSON object that is created for each incident and is used to store structured results from the integration commands and automation scripts. The context keys are strings and the values can be strings, numbers, objects, and arrays.
 
-The main use of context is to pass data between playbook tasks. Another usage is to capture the important structured data from automations and display the data in the incident summary.
-
-Since context is used within playbooks, it is critical that the context outputs remain reproducible. Meaning, that the structure of the context does not change. 
+The main use of context is to pass data between playbook tasks, one task stores output in the context and the other task reads that output from the context and uses it.
```
Comment (Itay4) [inline docs/context_and_ouputs/README.MD]: maybe add a screenshot of task input/output here?

File: docs/context_and_ouputs/README.MD [hunk 2]
```diff
@@ -1,126 +1,134 @@
...
 ```
 
-## DT (Demisto Transform Language)
-In the above example, we observe the entry context using ```(val.Address==obj.Address)```. This works to *tie together* related entry context objects. In this instance, we are using the value of the Address key as the unique identifier to search through the existing context and link related objects. This prevents data from being overwritten as well as further enriches an existing entry with more information. Learn more about linking context [here](https://github.com/demisto/content/blob/master/docs/code_conventions/README.MD#linking-context).
-
-
-## Adding Context to an Integration
-When adding context to an integration, we must explicitly define what the context paths are so they will be accessible. See below to understand how context should be defined in the Integration Settings.
+#### Integration YML
+In Demisto integration yml we will define outputs in the following way:
```
Comment (Itay4) [inline docs/context_and_ouputs/README.MD]:
```suggestion
In the integration YAML file, the command outputs are defined as such:
```

File: docs/context_and_ouputs/README.MD [hunk 3]
```diff
@@ -1,126 +1,134 @@
...
-
-AWS.S3.Buckets.Policy.Version
-AWS.S3.Buckets.Policy.PolicyId
-
 
-Centreon.Service.Name
-Centreon.Service.ServiceId
-Centreon.Service.Description
-```
+**Note**: The code **must** match the yml! The brand name (e.g. `ThreatStream`), the object name (e.g. `Analysis`)
```
Comment (Itay4) [inline docs/context_and_ouputs/README.MD]: not sure its clear whats youre trying to say here

File: docs/faq/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# FAQ
```
Comment (Itay4) [inline docs/faq/README.md]: need to add link to this

File: docs/faq/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+### Can I develop in JavaScript?
+Demisto supports JavaScript, but for contributions we require Python.
+
+### Which OSs are supported for development?
+Our recommended OS for development is either macOS or Linux, as we use bash and docker in some of our validation/testing flows.
+
+If you are working on Windows, you can either work with a Linux VM or utilize [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+
+### Licensing
+Demisto content repository is BSD license
```
Comment (Itay4) [inline docs/faq/README.md]: lets add link to the license file

File: docs/faq/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+### Which kind of contributions do you support?
+We support **Demisto Certified** and **Developer Supported**. For more details you can read [here]()
+
+### Which IDEs do you support?
+- We have PyCharm plugin [link]()
+- You can develop integrations and scripts in Demisto
+- Playbooks you must develop in Demisto
+
+### Which Python version you support?
+Demisto supports both Python2 and Python3, but for contributions we require Python3
```
Comment (Itay4) [inline docs/faq/README.md]: lets take the answer for this from @glicht PR

File: docs/faq/README.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+### Can I develop in JavaScript?
+Demisto supports JavaScript, but for contributions we require Python.
```
Comment (Itay4) [inline docs/faq/README.md]: lets take the answer for this from @glicht PR

File: docs/faq/README.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# FAQ
+
+### Which kind of contributions do you support?
+We support **Demisto Certified** and **Developer Supported**. For more details you can read [here]()
+
+### Which IDEs do you support?
+- We have PyCharm plugin [link]()
+- You can develop integrations and scripts in Demisto
+- Playbooks you must develop in Demisto
```
Comment (Itay4) [inline docs/faq/README.md]:
```suggestion
- Playbooks can only be developed in Demisto.
```

**Comment by anara123** [comment]
@fvigo FYI

File: docs/context_and_ouputs/README.MD [hunk 9]
```diff
@@ -1,126 +1,134 @@
...
-Centreon.Service.Description
-```
+**Note**: The code **must** match the context path in the yml
 
-In the context menu, you will see three fields; Context Path, Description and Type. Their uses are as follows:
-* **Context Path** - This is a Dot Notation representation of the path to access the context
-* **Description** - A short description of what the context is
-* **Type** - Indicating the type of value that is located at the path enables Demisto to format the data correctly
+### DT (Demisto Transform Language)
+In the above example, we observe the entry context using ```(val.Address==obj.Address)```. This works to *tie together* related entry context objects. In this instance, we are using the value of the Address key as the unique identifier to search through the existing context and link related objects. This prevents data from being overwritten as well as further enriches an existing entry with more information. Learn more about linking context [here](https://github.com/demisto/content/blob/master/docs/DT).
```
Comment (yaakovi) [inline docs/context_and_ouputs/README.MD]:
```suggestion
In the above example, we observe the entry context using ```(val.ReportID == obj.ReportID)```. This works to *tie together* related entry context objects. In this instance, we are using the value of the ReportID key as the unique identifier to search through the existing context and link related objects. This prevents data from being overwritten as well as further enriches an existing entry with more information. Learn more about linking context [here](https://github.com/demisto/content/blob/master/docs/DT).
```

**Comment by glicht** [comment]
@anara123 can we close this? It looks like it has been transferred to content-docs repo.

**Comment by anara123** [comment]
Yes we can

**Pull request closed without merging**